setup.sh

```diff
@@ -42,11 +42,21 @@ function _unset_vars
   unset SCRIPT USING_SELINUX
 }
 
-function _get_current_directory
+function _get_script_path
```

Comment:
This should, from my PoV be a single function call together with `_get_script_directory`. I find the introduction of a new variable, which is not even local (because it cannot be) or `unset` in then end rather misleading.

Reply:
I thought about this too. But decided to leave it as it is, for the mantra "one function does one thing". That way, $SCRIPT_PATH could also be reused if needed (in the future).

Reply:
Alright